src/runtime/slot-consumer.ts

```diff
@@ -15,20 +15,23 @@ import {Description} from './description.js';
 import {SlotConnection} from './recipe/slot-connection.js';
 import {HostedSlotContext, ProvidedSlotContext, SlotContext} from './slot-context.js';
 
-export interface Model {
-  id: string;
-}
-
 export interface Content {
   templateName?: string | Map<string, string>;
-  model?: {models: Model, hash: string};
+  model?: {models: any, hash: string};
   descriptions?: Map<string, Description>;
-  template?: string | Map<string, {}>;
+  template?: string | Map<string, string>;
 }
 
-export interface Render {
-  container?: {};
-  model?: {items: {models: Model[]}, subId: {}, message: {}};
+export interface Rendering {
+  // The 'parent' or owning object in the UI.
+  // TODO(jopra): At some point we should write an interface for this.
+  // tslint:disable-next-line: no-any
+  container?: any;
+  // The data to be used in templating.
+  // tslint:disable-next-line: no-any
+  model?: any;
```

Comment:
`{}` is probably better than `any`? it is definitely an object, we just don't know what's inside it.

Reply:
{} has no fields though. We currently are required to 'trust' that the programmer knows the fields exist.